Orange/widgets/visualize/owscatterplot.py

```diff
@@ -380,8 +380,14 @@ def update_graph(self, attributes=None, reset_view=True, **_):
         self.graph.zoomStack = []
         if attributes and len(attributes) == 2:
             self.attr_x, self.attr_y = attributes
+        X = BasicStats(self.data[:, self.attr_x], self.attr_x)
+        Y = BasicStats(self.data[:, self.attr_y], self.attr_y)
+        if X.nans or Y.nans:
+            self.warning("Some attributes contain missing values") 
+        else:
+             self.warning()      
         if not self.graph.have_data:
-            return
+            return    
```

Comment:
You should set your code editor to [strip trailing whitespace](https://www.google.com/search?q=strip+trailing+whitespace&tbm=isch) characters, or, at least, not introduce new ones.